### PR TITLE
Feature/251/performance retry timeout

### PR DESCRIPTION
**Pull request by tnobody**
This PR removes unnecessary retry mechanism from `AccessorUtils` and decreases the default timeout to 3s. 

It also introduces two new DSL functions `_setFetchTimeout` and `_getFetchTimeout`  to Sakuli to configure the this timeout manually.

First tests showed me that:
- Fail fast behavior feels more natural (if an element doesn't exists, the Error is throw much earlier)
- The Error message that is shown in output is much better to read, since it prints the "real" error from sakuli (with information about the queried Element) and not any cryptical "timeout exceeded message".

### CODE REVIEW

**Comment by ZNicoZ** [comment]
A manual _setFetchTimeout always results in a +4 seconds runtime:
image

Apart from that it looks fine and feels faster :)

**Comment by tnobody** [comment]
> Time offset with a manual timeout (see comment)

Could you provide, the testcase code and used parameters you used to observe this behavior?

Thanks in Advance.

**Comment by tnobody** [comment]
I ran an example testsuite and found out that the +4s offset is caused by the `_navigateTo`:

```typescript

(async () => {
    const testCase = new TestCase("My Typescript based test");
    try {
        await _navigateTo("https://sakuli.io");
        testCase.endOfStep("TS Open Landing Page", 5); //CHROME: 4.97s | FF: 5.08s
        await _click(_link("Getting started"));
        testCase.endOfStep("TS Navigate to Getting Started", 3); // CHROME: 0.22s | FF: 0.22s
        _setFetchTimeout(10_000);
        await _highlight(_code("npx init")); // CHROME: 10.03s | FF: 10.39s
        testCase.endOfStep("Find npm init code sample");
    } catch (e) {
        testCase.handleException(e);
    } finally {
        testCase.saveResult();
    }
})().then(done);

```

File: packages/sakuli-legacy/src/context/sahi/sahi-api.interface.ts [hunk 1]
```diff
@@ -14,4 +14,17 @@ export interface SahiApi extends ActionApi, AccessorApi, RelationApi, FetchApi,
...
+     * @param timeOutMs - Timeout in milliseconds
+     */
+    _setFetchTimeout(timeOutMs: number): void;
+
+    /**
+     * Gets the maximum time Sakuli will try to find an element on dom.
+     *
+     * @param timeOutMs - Timeout in milliseconds
+     */
+    _getFetchTimeout(timeOutMs: number): void;
```
Comment (cliffle) [inline packages/sakuli-legacy/src/context/sahi/sahi-api.interface.ts]:
```suggestion
    _getFetchTimeout(): number;
```
There is no parameter for `_getFetchTimeout()`

Comment (tnobody) [reply packages/sakuli-legacy/src/context/sahi/sahi-api.interface.ts]: Fixed in 8bc1bf2

**Pull request merged**